### PR TITLE
[configure-splash-screen] <docs>: update usage description

### PR DESCRIPTION
**Pull request by bbarthec**
- added small CHANGELOG entry
- updated README with the actual command behaviour & options description

Follow-up of https://github.com/expo/expo-cli/pull/2297

# TODOs:
- [ ] publish `@expo/configure-splash-screen`
- [ ] update dependency in `expo-splash-screen` & add CHANGELOG entry
- [ ] publish `expo-splash-screen`

### CODE REVIEW

File: packages/configure-splash-screen/README.md [hunk 1]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--status-bar-style <style>` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
+- `--status-bar-hidden` - Hides the StatusBar.
+- `--status-bar-background-color <color>` - (only for Android platform) Customizes the background color of the StatusBar. Accepts a valid color (see 'background-color' supported formats).
+- `--status-bar-translucent` - (only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).
+- `--dark-mode-status-bar-style <style>` - (only for Android platform) The very same as 'status-bar-style' option, but applied only in dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.
+- `--dark-mode-status-bar-background-color <color>` - (only for Android platform) The very same as 'status-bar-background-color', but applied only in the dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:59]:
```suggestion
- `--dark-mode-status-bar-background-color <color>` - (only for Android platform) The same as 'status-bar-background-color', but applied only in the dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.
```

File: packages/configure-splash-screen/README.md [hunk 2]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
+- `--status-bar-style <style>` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
+- `--status-bar-hidden` - Hides the StatusBar.
+- `--status-bar-background-color <color>` - (only for Android platform) Customizes the background color of the StatusBar. Accepts a valid color (see 'background-color' supported formats).
+- `--status-bar-translucent` - (only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).
+- `--dark-mode-status-bar-style <style>` - (only for Android platform) The very same as 'status-bar-style' option, but applied only in dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:58]:
```suggestion
- `--dark-mode-status-bar-style <style>` - (only for Android platform) The same as 'status-bar-style', but applied only in dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.
```

File: packages/configure-splash-screen/README.md [hunk 3]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
-- `--statusbar-hidden` - Hides the StatusBar.
-- `--statusbar-background-color [statusBarBackgroundColor]` - (only for Android platform) Customizes the background color of the StatusBar. Valid css-formatted color (see backgroundColor supported formats).
-- `--dark-mode-statusbar-background-color [darkModeStatusBarBackgroundColor]` - (only for Android platform) The very same as 'statusbar-background-color' option, but applied only in dark mode. Available only if `statusbar-style` is provided.
-- `--statusbar-translucent` - (only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).
+- `-p, --platform <platform>` - Selected platform to configure. Available values: "android" | "ios" | "all" (default: "all").
+- `-b, --background-color <color>` - (required) Valid css-formatted color (hex (#RRGGBB[AA]), rgb[a], hsl[a], named color (https://drafts.csswg.org/css-color/#named-colors)) that would be used as the background color for native splash screen view.
+- `-i, --image-path <path>` - Path to valid .png image that will be displayed in native splash screen.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:50]:
```suggestion
- `-i, --image-path <path>` - Path to valid .png image that will be displayed on the splash screen.
```

File: packages/configure-splash-screen/README.md [hunk 4]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `-r, --image-resize-mode <resizeMode>` - ResizeMode to be used for native splash screen image. Available only if 'image-path' is provided as well. Available values: "contain" | "cover" | "native" (only available for android platform)) (default: "contain"). See [resize modes](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#built-in-splash-screen-image-resize-modes) for more information.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:51]:
```suggestion
- `-r, --image-resize-mode <resizeMode>` - Resize mode to be used for the splash screen image. Available only if 'image-path' is provided as well. Available values: "contain" | "cover" | "native" ("native" is only available for Android)) (default: "contain"). See [resize modes](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#built-in-splash-screen-image-resize-modes) for more information.
```

File: packages/configure-splash-screen/README.md [hunk 5]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--dark-mode-background-color <color>` - Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode. Providing this option enables other dark-mode related options.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:52]:
```suggestion
- `--dark-mode-background-color <color>` - Color (see 'background-color' supported formats) that would be used as the background color for the splash screen in dark mode. Providing this option enables other dark-mode related options.
```

File: packages/configure-splash-screen/README.md [hunk 6]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:53]:
```suggestion
- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed on the splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
```

File: packages/configure-splash-screen/README.md [hunk 7]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--status-bar-style <style>` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
+- `--status-bar-hidden` - Hides the StatusBar.
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:55]:
```suggestion
- `--status-bar-hidden` - Hides the status bar.
```

File: packages/configure-splash-screen/README.md [hunk 8]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--status-bar-background-color <color>` - (only for Android platform) Customizes the background color of the StatusBar. Accepts a valid color (see 'background-color' supported formats).
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:56]:
```suggestion
- `--status-bar-background-color <color>` - (only for Android platform) Customizes the background color of the status bar. Accepts a valid color (see 'background-color' supported formats).
```

File: packages/configure-splash-screen/README.md [hunk 9]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `-i, --image-path <path>` - Path to valid .png image that will be displayed in native splash screen.
+- `-r, --image-resize-mode <resizeMode>` - ResizeMode to be used for native splash screen image. Available only if 'image-path' is provided as well. Available values: "contain" | "cover" | "native" (only available for android platform)) (default: "contain"). See [resize modes](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#built-in-splash-screen-image-resize-modes) for more information.
+- `--dark-mode-background-color <color>` - Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode. Providing this option enables other dark-mode related options.
+- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
+- `--status-bar-style <style>` - Customizes the color of the StatusBar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
```
Comment (brentvatne) [inline packages/configure-splash-screen/README.md:54]:
```suggestion
- `--status-bar-style <style>` - Customizes the color of the status bar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
```

Comment (brentvatne) [reply packages/configure-splash-screen/README.md:54]: not necessary for this PR but maybe we should change "light-content" and "dark-content" to just "light" and "dark" like in https://docs.expo.io/versions/latest/sdk/status-bar/#statusbarstyle

**Review by brentvatne — APPROVED** [review]
looks good, left some small notes to improve wording in the descriptions. we should change those in cli-command as well

File: packages/configure-splash-screen/src/cli-command.ts [hunk 10]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
+      )} (default: "${Platform.ALL}").`
     )
     .option(
       '-b, --background-color <color>',
-      `Valid css-formatted color (hex (#RRGGBB[AA]), rgb[a], hsl[a], named color (https://drafts.csswg.org/css-color/#named-colors)) that would be used as the background color for native splash screen view.`
+      `(required) Valid css-formatted color (hex (#RRGGBB[AA]), rgb[a], hsl[a], named color (https://drafts.csswg.org/css-color/#named-colors)) that would be used as the background color for native splash screen view.`
     )
     .option(
       '-i, --image-path <path>',
       'Path to valid .png image that will be displayed in native splash screen.'
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:151]:
```suggestion
      'Path to valid .png image that will be displayed on the splash screen.'
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 11]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
     )
     .option(
       '-r, --image-resize-mode <resizeMode>',
-      `ResizeMode to be used for native splash screen image. Available values: ${getAvailableOptions(
+      `ResizeMode to be used for native splash screen image. Available only if 'image-path' is provided as well. Available values: ${getAvailableOptions(
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:155]:
```suggestion
      `Resize mode to be used for the splash screen image. Available only if 'image-path' is provided as well. Available values: ${getAvailableOptions(
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 12]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
         SplashScreenImageResizeMode
-      )} (only available for android platform)).`
+      )} (only available for android platform)) (default: "${
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:157]:
```suggestion
      )} ("native" is only available for Android)) (default: "${
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 13]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
+        SplashScreenImageResizeMode.CONTAIN
+      }").`
     )
     .option(
       '--dark-mode-background-color <color>',
-      `Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode.`
+      `Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode. Providing this option enables other dark-mode related options.`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:163]:
```suggestion
      `Color (see 'background-color' supported formats) that would be used as the background color for the splash screen in dark mode. Providing this option enables other dark-mode related options.`
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 14]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
     )
     .option(
       '--dark-mode-image-path <path>',
-      'Path to valid .png image that will be displayed in native splash screen in dark mode only.'
+      `Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.`
     )
     .option(
       '--status-bar-style <style>',
       `Customizes the color of the StatusBar icons. Available values: ${getAvailableOptions(
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:171]:
```suggestion
      `Customizes the color of the status bar icons. Available values: ${getAvailableOptions(
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 15]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
         SplashScreenStatusBarStyle
-      )}.`
+      )} (default: "${SplashScreenStatusBarStyle.DEFAULT}").`
     )
     .option('--status-bar-hidden', `Hides the StatusBar.`)
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:175]:
```suggestion
    .option('--status-bar-hidden', `Hides the status bar.`)
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 16]
```diff
@@ -183,13 +183,13 @@ export default () =>
     )
     .option(
       '--dark-mode-status-bar-style <style>',
-      `(only for Android platform) The very same as 'statusbar-style' option, but applied only in dark mode. Available only if 'statusbar-style' is provided.`
+      `(only for Android platform) The very same as 'status-bar-style' option, but applied only in dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:186]:
```suggestion
      `(only for Android platform) The same as 'status-bar-style', but applied only in dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.`
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 17]
```diff
@@ -183,13 +183,13 @@ export default () =>
...
     )
     .option(
       '--dark-mode-status-bar-background-color <color>',
-      `(only for Android platform) The very same as 'status-bar-background-color', but applied only in the dark mode. Available only if 'statusbar-style' is provided.`
+      `(only for Android platform) The very same as 'status-bar-background-color', but applied only in the dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:190]:
```suggestion
      `(only for Android platform) The same as 'status-bar-background-color', but applied only in the dark mode. Available only if 'dark-mode-background-color' and 'status-bar-style' are provided as well.`
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 18]
```diff
@@ -142,35 +138,39 @@ export default () =>
...
     )
     .option(
       '--dark-mode-background-color <color>',
-      `Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode.`
+      `Color (see 'background-color' supported formats) that would be used as the background color for native splash screen in dark mode. Providing this option enables other dark-mode related options.`
     )
     .option(
       '--dark-mode-image-path <path>',
-      'Path to valid .png image that will be displayed in native splash screen in dark mode only.'
+      `Path to valid .png image that will be displayed in native splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:167]:
```suggestion
      `Path to valid .png image that will be displayed on the splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.`
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 19]
```diff
@@ -142,37 +138,41 @@ export default () =>
...
+      `Customizes the color of the status bar icons. Available values: ${getAvailableOptions(
         SplashScreenStatusBarStyle
-      )}.`
+      )} (default: "${SplashScreenStatusBarStyle.DEFAULT}").`
     )
-    .option('--status-bar-hidden', `Hides the StatusBar.`)
+    .option('--status-bar-hidden', `Hides the status bar.`)
     .option(
       '--status-bar-background-color <color>',
       `(only for Android platform) Customizes the background color of the StatusBar. Accepts a valid color (see 'background-color' supported formats).`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:178]:
```suggestion
      `(only for Android platform) Customizes the background color of the status bar. Accepts a valid color (see 'background-color' supported formats).`
```

File: packages/configure-splash-screen/src/cli-command.ts [hunk 20]
```diff
@@ -142,54 +138,58 @@ export default () =>
...
-    .option('--status-bar-hidden', `Hides the StatusBar.`)
+    .option('--status-bar-hidden', `Hides the status bar.`)
     .option(
       '--status-bar-background-color <color>',
-      `(only for Android platform) Customizes the background color of the StatusBar. Accepts a valid color (see 'background-color' supported formats).`
+      `(only for Android platform) Customizes the background color of the status bar. Accepts a valid color (see 'background-color' supported formats).`
     )
     .option(
       '--status-bar-translucent',
       `(only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).`
```
Comment (bbarthec) [inline packages/configure-splash-screen/src/cli-command.ts:182]:
```suggestion
      `(only for Android platform) Makes the status bar translucent (enables drawing under the StatusBar area).`
```

File: packages/configure-splash-screen/README.md [hunk 21]
```diff
@@ -32,32 +33,30 @@ You can use it to configure your native iOS and Android project according to you
...
+- `--dark-mode-background-color <color>` - Color (see 'background-color' supported formats) that would be used as the background color for the splash screen in dark mode. Providing this option enables other dark-mode related options.
+- `--dark-mode-image-path <path>` - Path to valid .png image that will be displayed on the splash screen in dark mode only. Available only if 'dark-mode-background-color' is provided as well.
+- `--status-bar-style <style>` - Customizes the color of the status bar icons. Available values: "default" | "light-content" | "dark-content" (default: "default").
+- `--status-bar-hidden` - Hides the status bar.
+- `--status-bar-background-color <color>` - (only for Android platform) Customizes the background color of the status bar. Accepts a valid color (see 'background-color' supported formats).
+- `--status-bar-translucent` - (only for Android platform) Makes the StatusBar translucent (enables drawing under the StatusBar area).
```
Comment (bbarthec) [inline packages/configure-splash-screen/README.md:57]:
```suggestion
- `--status-bar-translucent` - (only for Android platform) Makes the status bar translucent (enables drawing under the status bar area).
```

**Pull request merged**